ALL_DB_BACKUP_THU.sql

```diff
@@ -0,0 +1,17400 @@
+--
+-- PostgreSQL database cluster dump
+--
+
+SET default_transaction_read_only = off;
+
+SET client_encoding = 'UTF8';
+SET standard_conforming_strings = on;
+
+--
+-- Roles
+--
+
+CREATE ROLE clide;
+ALTER ROLE clide WITH NOSUPERUSER INHERIT NOCREATEROLE NOCREATEDB LOGIN NOREPLICATION NOBYPASSRLS;
+CREATE ROLE clideadmin;
+ALTER ROLE clideadmin WITH NOSUPERUSER INHERIT NOCREATEROLE NOCREATEDB LOGIN NOREPLICATION NOBYPASSRLS;
+CREATE ROLE clidegui;
+ALTER ROLE clidegui WITH NOSUPERUSER INHERIT NOCREATEROLE NOCREATEDB LOGIN NOREPLICATION NOBYPASSRLS;
+CREATE ROLE datacomp;
+ALTER ROLE datacomp WITH NOSUPERUSER INHERIT NOCREATEROLE NOCREATEDB LOGIN NOREPLICATION NOBYPASSRLS;
+CREATE ROLE postgres;
```

Comment:
@isedwards 

I manually ran the backup script. This is the resulting file. I tried to load it as database init script in docker. But it failed at this line as the postgres role was already there. I looked for option so that these roles are not created when we dump the data. But I found no such option.